### PR TITLE
devices: Add qcom sm8{1,2,3}50-mtp devices

### PR DESCRIPTION
**Pull request by alimon**
Signed-off-by: Aníbal Limón <anibal.limon@linaro.org>

### CODE REVIEW

**Comment by alimon** [comment]
@roxell Can you take a look?, are new qcom devices same as sdm845-mtp.

File: devices/sm8150-mtp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{% set ROOTFS_URL_COMP = ROOTFS_URL_COMP|default("gz") %}
+
+{% set pre_boot_command = false %}
+{% set pre_power_command = false %}
+{% set pre_os_command = false %}
+{% set rootfs_label = 'userdata' %}
+
+{% block device_type %}sm8150-mtp{% endblock %}
+
+{% block auto_login_commands %}
```
Comment (roxell) [inline devices/sm8150-mtp:14]: This block should already be included from include/fastboot.jinja2 that gets included from projects/lt-qcom/fastboot.jinja2

File: devices/sm8250-mtp [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+{% set ROOTFS_URL_COMP = ROOTFS_URL_COMP|default("gz") %}
+
+{% set pre_boot_command = false %}
+{% set pre_power_command = false %}
+{% set pre_os_command = false %}
+{% set rootfs_label = 'userdata' %}
+
+{% block device_type %}sm8250-mtp{% endblock %}
+
+{% block auto_login_commands %}
```
Comment (roxell) [inline devices/sm8250-mtp:14]: same here

File: devices/sm8350-mtp [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+{% set ROOTFS_URL_COMP = ROOTFS_URL_COMP|default("gz") %}
+
+{% set pre_boot_command = false %}
+{% set pre_power_command = false %}
+{% set pre_os_command = false %}
+{% set rootfs_label = 'userdata' %}
+
+{% block device_type %}sm8350-mtp{% endblock %}
+
+{% block auto_login_commands %}
```
Comment (roxell) [inline devices/sm8350-mtp:14]: same here

**Comment by roxell** [comment]
can we add the new devices with some tests here.
.github/workflows/test-plans-pipeline.yml

File: devices/sm8150-mtp (hunk 1 above)
Comment (alimon) [reply devices/sm8150-mtp:14]: @roxell Ack, older template also fixed previous qcom boads.

File: devices/sm8350-mtp (hunk 3 above)
Comment (alimon) [reply devices/sm8350-mtp:14]: Ack

File: devices/sm8250-mtp (hunk 2 above)
Comment (alimon) [reply devices/sm8250-mtp]: Ack.

**Comment by alimon** [comment]
> can we add the new devices with some tests here. .github/workflows/test-plans-pipeline.yml

Done

**Review by nareshkamboju — APPROVED** [review]

**Pull request merged**